arches_querysets/models.py

```diff
@@ -377,35 +362,48 @@ def sync_private_attributes(self, source):
         self._queried_nodes = source._queried_nodes
         self._permitted_nodes = source._permitted_nodes
 
-    def create_blank_tile(self, nodegroup, parent_tile=None):
-        children = (
-            nodegroup.children.all()
-            if arches_version >= (8, 0)
-            else nodegroup.nodegroup_set.all()
+    def append_tile(self, nodegroup_alias):
```

Comment:
This is golden if appending to cardinality_n nodegroup, but if I try with a cardinality 1 nodegroup that already has a tile, I only get a TypeError:

> (Pdb) self.semantic_resource_none.append_tile("datatypes_1")
> *** TypeError

Is it a heavy lift to throw an error that's a little more intelligible?